### ISSUE TITLE
Solution for User-agent 403 errors

### ISSUE THREAD

**Issue by njb-said** [MC-API/Issues#29]
After a conclusion on #17, I am going to:
- Mess around with cloudflare (the culprit) to fix it or disable the security setting entirely
- Add a note to the [docs](https://github.com/mc-api/static) to tell users to make sure they have a valid user agent (not ideal)

I am also going to rewrite the java client for mc-api to be more diverse and lightweight (hopefully)
You can find the java client [here](https://github.com/mc-api/uuid-java).
And the documentation (you can contribute) [here](https://github.com/mc-api/static)

tl;dr - You need to make sure your API requests have a valid User-agent field otherwise cloudflare blocks it.

**Comment by njb-said** [comment]
I have edited cloudflare's settings, I am going to monitor it for a few days.

**Comment by ImNatho** [comment]
Follow up: After trying a few days I have failed to get any 403 errors from the US or EU servers. Looks like changing the user-challenge has solved this.
